__tests__/integration/api.test.js

```diff
@@ -299,6 +299,49 @@ describe("/api/articles", () => {
         expect(body.msg).toBe("Bad Request");
       });
   });
+
+  test("GET: 200 returns a list of articles filtered by topic", () => {
+    const queryTopic = "mitch";
+    const expectedLength = testData.articleData.filter(
+      (article) => article.topic === queryTopic
+    ).length;
+    return request(app)
+      .get("/api/articles")
+      .query({ topic: queryTopic })
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        const isCorrectTopic = (article) => article.topic === queryTopic;
+        expect(articles).toHaveLength(expectedLength);
+        expect(articles).toSatisfyAll(isCorrectTopic);
+      });
+  });
+
+  test("GET: 200 returns an empty array when no articles match topic query", () => {
```

Comment:
we want a 404 when the topic doesn't exist and a 200 when the topic does exist but no associated articles, I think if you look in the test data `paper` is one of those